ckanext/charts/chart_builders/base.py

```diff
@@ -607,6 +607,14 @@ def height_field(self) -> dict[str, Any]:
             "group": "Data",
         }
 
+    def more_info_button(self) -> dict[str, Any]:
```

Comment:
follow the name format of other methods -> `more_info_button_field`
and add some docstring to describe what is it for